release/models/platform/openconfig-platform-fabric.yang

```diff
@@ -0,0 +1,88 @@
+module openconfig-platform-fabric {
+
+  yang-version "1";
+
+  // namespace
+  namespace "http://openconfig.net/yang/platform/fabric";
+
+  prefix "oc-fabric";
+
+  import openconfig-platform { prefix oc-platform; }
+  import openconfig-platform-types { prefix oc-platform-types; }
+  import openconfig-extensions { prefix oc-ext; }
+
+
+  // meta
+  organization "OpenConfig working group";
+
+  contact
+    "OpenConfig working group
+    www.openconfig.net";
+
+  description
+    "This module defines data related to FABRIC components in
+    the openconfig-platform model";
+
+  oc-ext:openconfig-version "0.1.0";
+
+  revision "2022-04-20" {
+    description
+      "Initial revision";
+    reference "0.1.0";
+  }
+
+  // OpenConfig specific extensions for module metadata.
+  oc-ext:regexp-posix;
```

Comment:
I believe that this can be removed now -- we've made all the patterns have their own extension for POSIX patterns, and this module has none within it.

Reply:
Thanks for the catch. I'll remove this from both files -- openconfig-platform-fabric.yang and openconfig-platform-controller-card.yang